### DISCUSSION TITLE
After installing backpack-pro all assets file path changed so not loading correctly

### DISCUSSION THREAD

**Discussion by shilpitrusty** [Laravel-Backpack/community-forum#917] [Q&A (Help)] [answered]
After installing backpack-pro all assets file path changed so not loading correctly. Can anyone help me with this so that I can change assets path?
I am using latest vesion of laravel and backpack

**Comment by pxpm** [comment]
I don't fully get your questions
PRO does not change file paths at all, or assets. 

I would advise you to run `php artisan basset:check` to make sure you have backpack properly installed. 

Post some screenshots of your errors to help us find the issue. 

Cheers

**Reply by shilpitrusty** [reply]
![image](https://github.com/Laravel-Backpack/community-forum/assets/166003001/c165ec11-2cca-4965-acce-922b8197511e)
It gives this error

**Reply by pxpm** [reply]
Hey @shilpitrusty this is not a backpack specific issue.

You don't have the webserver properly configured. In xampp you should create a vhost that points to that public folder index.php, but you should access your project with a url like `http://my-project.test` 

I would recommend you follow some tutorial first on how to setup Laravel using xampp like https://gist.github.com/bradtraversy/7485f928e3e8f08ee6bccbe0a681a821?permalink_comment_id=5000359#gistcomment-5000359

Or you can use a different development tool like Laragon https://laragon.org/index.html (I am not affiliated with them, they are a good project that works well and I use it to develop on windows). 

Once you have laravel up and running properly, this will be a non issue.

Cheers

**Reply by karandatwani92** [reply, marked as answer]
@shilpitrusty 
OR, you can run `php artisan serve` in your project directory and access your app at `http://localhost:8000`. 

If going this way, make sure to set `APP_URL=http://localhost:8000`
